[2024-03-15 17:48:09,314] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T17:47:00+00:00 [queued]>
[2024-03-15 17:48:09,336] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-15T17:47:00+00:00 [queued]>
[2024-03-15 17:48:09,336] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 17:48:09,336] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 17:48:09,337] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 17:48:09,348] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-15T17:47:00+00:00
[2024-03-15 17:48:09,356] {standard_task_runner.py:52} INFO - Started process 38054 to run task
[2024-03-15 17:48:09,358] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-15T17:47:00+00:00', '--job-id', '3072', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpv6cglzsq', '--error-file', '/tmp/tmpirtj8r05']
[2024-03-15 17:48:09,359] {standard_task_runner.py:77} INFO - Job 3072: Subtask compute_rf_score
[2024-03-15 17:48:09,406] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-15T17:47:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 17:48:09,475] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T17:47:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T17:47:00+00:00
[2024-03-15 17:48:10,623] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 17:48:10,636] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240315T174700, start_date=20240315T174809, end_date=20240315T174810
[2024-03-15 17:48:10,676] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-15 17:48:10,697] {local_task_job.py:151} INFO - Task exited with return code 0
